src/storage/storage.ts

```diff
@@ -1,3 +1,6 @@
+import { isObject } from '../isObject/isObject';
+import { isString } from '../isString/isString';
```

Comment:
Is is okay to import helpers from other parts of libs?

Reply:
the same pattern seems to be used in other places: https://github.com/guardian/libs/blob/1912df53e54ee231539353746d554287a7e59013/src/locale/getLocale.ts#L4